text/0044-heartbeat.md

```diff
@@ -0,0 +1,69 @@
+- Start Date: 2022-12-09
+- RFC Type: feature
+- RFC PR: N/A
+- RFC Status: draft
+
+# Summary
+
+Introduce an infrastructure monitoring agent in all SDKs supported by us. This agent will be triggered every time the user launches the application and will send us information about the state of their integration with Sentry.
```

Comment:
> send us information about the state of their integration

Does "the state of their integration" refer to the configuration of the SDK? What information are we looking for specifically?

Reply:
state means:

On -> The connection of the user's SDK with Sentry was established
Off -> There is no connection between the user's SDK and Sentry

Reply:
@iker-barriocanal for now it will be rather simple - boolean initialized or not basically

but in the future that could scale and be dependent on platform. Thinking of scaling serverside services and their "alive" status or time alive. Versus client side where that type feature would not be so interesting. Every browser opened for example. But we are getting past scope of the initial phase, But I think you were correct above it is important to consider when determining what event makes sense for future planning of the payload.